src/services/room-service.js

```diff
@@ -117,6 +117,13 @@ export default class RoomService {
     return updatedRoom;
   }
 
+  async updateRoomDocuments(roomId, { documents }) {
+    await this.roomStore.updateRoomDocuments(roomId, { documents });
+    const updatedRoom = await this.roomStore.getRoomById(roomId);
+
+    return updatedRoom;
+  }
```

Comment:
I know it would add more complexity but I think we should ensure data integrity here by
* Get a room lock
* Fetch the current room
* Check that the new documents are still exactly the same as the old ones, just in different order
* Save the updated room
* Release the lock

Reply:
That's true, we should ensure valid document ids here, not just id-type values.
I included this check, thanks!

Reply:
Great, thanks :+1: